vars/checkChangeInDevelopment.groovy

```diff
@@ -0,0 +1,89 @@
+import com.sap.piper.GitUtils
+import groovy.transform.Field
+
+import com.sap.piper.ConfigurationMerger
+import com.sap.piper.cm.ChangeManagement
+import com.sap.piper.cm.ChangeManagementException
+
+@Field def STEP_NAME = 'checkChangeInDevelopment'
+
+@Field Set parameterKeys = [
+    'credentialsId',
+    'endpoint',
+    'failIfStatusIsNotInDevelopment',
+    'git_from',
+    'git_to',
+    'git_label',
+    'git_format'
+  ]
+
+@Field Set stepConfigurationKeys = [
+    'credentialsId',
+    'endpoint',
+    'failIfStatusIsNotInDevelopment',
+    'git_from',
+    'git_to',
+    'git_label',
+    'git_format'
+  ]
+
+def call(parameters = [:]) {
+
+    handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
+
+        System.err<<"PARAMS: ${parameters}\n"
```

Comment:
Must not be committed. Was only troubleshooting. Was introduced by me as an early work in progress.

Reply:
The file `checkChangeInDevelopment.groovy` should not be here. I forgot to remove it, I will do it. This should be part of your PR.